app/helpers/components_helper.rb

```diff
@@ -277,4 +278,12 @@ def form_cancel_button
     end
   end
 
+  def is_reused(ontology_uri_pattern:, concept_id:)
+    if ontology_uri_pattern[0] # if uriRegexPattern exists
```

Comment:
why you do not pass `@submission` instead of this array `ontology_uri_pattern` ?